tests/conftest.py

```diff
@@ -0,0 +1,176 @@
+import os
+import logging
+from pathlib import Path
+import pytest
+import yaml
+
+import simtools.config as cfg
+
+
+logger = logging.getLogger()
+
+
+def write_configuration_test_file(config_file, config_dict):
+    """
+    Write a simtools configuration file
+
+    Do not overwrite existing files
+    """
+
+    if not Path(config_file).exists():
+        with open(config_file, "w") as output:
+            yaml.safe_dump(
+                config_dict,
+                output,
+                sort_keys=False)
+
+
+@pytest.fixture
+def tmp_test_directory(tmpdir_factory):
+    tmp_test_dir = tmpdir_factory.mktemp("test-data")
+    tmp_sub_dirs = ['/resources/', '/output/', '/simtel/']
+    for sub_dir in tmp_sub_dirs:
+        tmp_sub_dir = tmp_test_dir + sub_dir
```

Comment:
Should probably use `join` here and then remove the `/` from the list above.

Reply:
Now that I continue reading, this entire file is not using `join`. I would vote to changing everywhere to `join` and casting to string after the command rather than before and using `+`. It's not critical though.

Reply:
Good point. Replaced all `+` by `join`

Reply:
Oh no - reading the next comment, I've replaced all by `/`